geoserverexplorer/test/testplugin.py

```diff
@@ -52,4 +54,5 @@ def runAllUnitTests():
     suite.addTest(deleteSuite())
     suite.addTest(dragdropSuite())
     suite.addTest(guiSuite())
+    suite.addTest(symbologySuite())
```

Comment:
yes but strange error now in the CI build, i'm a little bit lost with this => git not found